context.go

```diff
@@ -318,3 +269,50 @@ func contains(a []string, b string) bool {
 	}
 	return false
 }
+
+// getGOPATH returns the GOPATHs from the passed environment variables.
```

Comment:
nit: move these back up to their original location in order to minimize churn within the file